testutil/environment/environment.go

```diff
@@ -82,14 +82,21 @@ func getPlatformDefaults(info types.Info, osType string) PlatformDefaults {
 			ContainerStoragePath: toSlash(containersPath),
 		}
 	case "windows":
-		baseImage := "microsoft/windowsservercore"
+		var baseImage string
+		// Default to Windows Server 2019
+		baseImageRepo := "mcr.microsoft.com/windows/servercore"
+		baseImageTag := "1809"
+
 		if overrideBaseImage := os.Getenv("WINDOWS_BASE_IMAGE"); overrideBaseImage != "" {
-			baseImage = overrideBaseImage
-			if overrideBaseImageTag := os.Getenv("WINDOWS_BASE_IMAGE_TAG"); overrideBaseImageTag != "" {
-				baseImage = baseImage + ":" + overrideBaseImageTag
-			}
+			// When setting `WINDOWS_BASE_IMAGE`, assume it's a full reference, including tag
```

Comment:
Oh, this comment does not match how this variable is used in the `Dockerfile` :sweat_smile: